test/core/unreached-invalid.wast

```diff
@@ -249,6 +249,42 @@
   ))
   "type mismatch"
 )
+(assert_invalid
+  (module (func $type-unary-num-vs-void-in-block-after-unreachable
+    (unreachable) (block (drop (i32.eq (nop))))
```

Comment:
Isn't this the same as line 204?
Also, did you mean `eqz`?.

Reply:
Dropped this one.